server/routes/notification.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+
+const {
+  getAllNotifications,
+  createNotification,
+  markNotificationRead,
+  getUnreadNotifications
+} = require("../controllers/notification");
+
+router.route("/create").post(protect, createNotification);
+router.route("/mark/:notification_id").post(protect, markNotificationRead);
+router.route("/getAll").get(protect, getAllNotifications);
```

Comment:
Please use hyphenated case for route names.

Eg: `get-profiles`

Reply:
used hyphenated case